### ISSUE TITLE
how to compute adequacy score?

### ISSUE THREAD

**Issue by hyungkwonko** [PrithivirajDamodaran/Parrot#49]
Hi, thank you for open sourcing this.

Could you please let me know where can I find the training code of the adequacy score?
or is there anyway I can re-implement to calculate the adequacy score?

Thanks,

**Comment by damyana79** [comment]
Yes, I also have the same question. If I want to train a parrot model for a different language, how is adequacy computed?

**Comment by damyana79** [comment]
From the first glance it seems to be a model trained on an entailment task. Still, I'm not sure.
